ksqldb-engine/src/test/java/io/confluent/ksql/services/DefaultConnectClientTest.java

```diff
@@ -162,32 +159,14 @@ public void testCreate() throws JsonProcessingException {
   @Test
   public void testValidate() throws JsonProcessingException {
     // Given:
-    final String plugin = SAMPLE_PLUGIN.className();
+    final String plugin = SAMPLE_PLUGIN.getClassName();
     final String url = String.format(pathPrefix + "/connector-plugins/%s/config/validate", plugin);
     final ConfigInfos body = new ConfigInfos(
         plugin,
-        1,
-        ImmutableList.of("Common"),
-        ImmutableList.of(new ConfigInfo(new ConfigKeyInfo(
-            "file",
```

Comment:
I am not really familiar with the code but where did all these configs go? We don't need them anymore?